adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/UserCredential.cs

```diff
@@ -29,54 +29,35 @@
 
 namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
-    internal enum UserAuthType
-    {
-        IntegratedAuth,
-        UsernamePassword
-    }
+    using UserAuthType = Microsoft.Identity.Core.UserAuthType;
+
 
     /// <summary>
     /// Credential used for integrated authentication on domain-joined machines.
     /// </summary>
-    public class UserCredential
-    {
+    public class UserCredential: Microsoft.Identity.Core.UserCred {
```

Comment:
>UserCred [](start = 57, length = 8)

Not a fan of abbreviations, pls. use full name

Reply:
Fun facts: when I was composing the `Core.UserCred`, I did name it `UserCredential`. However it caused MANY existing, untouched code broken, due to many files use both the adal namespace and the core namespace and there would be an ambiguity.

Not a fan of being forced to fix hundreds of compiling error even before I started to write the first line of meaningful code, I chose a different name to get the prototype working asap. Now, at a hindsight, that was actually an efficient way even if we are still going to rename them now, because, now with a runnable code base, we can use the help of [Visual Studio rename refactoring](https://docs.microsoft.com/en-us/visualstudio/ide/reference/rename?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev15.query%3FappId%3DDev15IDEF1%26l%3DEN-US%26k%3Dk(vs.csharp.refactoring.rename)%3Bk(DevLang-csharp)%26rd%3Dtrue), which automatically reveals there would be "91 reference in 31 files" needs to be changed. #WorkingOnIt

Reply:
userCredential for local vars. Uppper case if it's a property

---
In reply to: [203567039](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1118#discussion_r203567039) [](ancestors = 203567039)